src/validation.h

```diff
@@ -609,6 +601,14 @@ class CChainState {
     CChainState(BlockManager& blockman) : m_blockman(blockman) {}
     CChainState();
 
+    /**
+    * Whether this chainstate is undergoing initial block download.
+    *
+    * Mutable because we need to be able to mark IsInitialBlockDownload()
+    * const, which latches this for caching purposes.
+    */
+    mutable std::atomic<bool> m_cached_finished_ibd{false};
+
```

Comment:
Why don't keep the variable private and add a function to set it?

Reply:
Probably sensible but this was done in a rush yesterday and this PR is to provide a temp solution to keep help stakers.